lib/platform/github/index.ts

```diff
@@ -1735,3 +1744,79 @@ export async function getVulnerabilityAlerts(): Promise<VulnerabilityAlert[]> {
   }
   return alerts;
 }
+
+async function pushFiles({
+  branchName,
+  files,
+  message,
+}: CommitFilesConfig): Promise<CommitSha | null> {
+  const additions: FileAddition[] = [];
+  const deletions: FileDeletion[] = [];
+
+  for (const file of files) {
+    if (file.type === 'addition') {
+      additions.push(file);
+    } else {
+      deletions.push(file);
+    }
+  }
+
+  const executableFiles = additions.filter(({ isExecutable }) => isExecutable);
+  if (executableFiles.length) {
+    logger.warn(
+      { branchName, executableFiles: executableFiles.map(({ path }) => path) },
+      'Platform-native commit: found executable files'
+    );
+    return null;
```

Comment:
Shouldn't we simply continue after warn? 🤷‍♂️

Reply:
Yes, let's warn but commit